.github/actions/build_setup/action.yml

```diff
@@ -28,9 +26,20 @@ runs:
       shell: bash
       run: chmod +x gradlew
 
+    - name: Cache build directory
+      uses: actions/cache@v4
+      with:
+        path: |
+          build/neoForge
+          build/neoForm
+          build/tmp/.cache/**
+        key: "${{ runner.os }}-gradle-${{ hashFiles('**/libs.versions.*', '**/*.gradle*', '**/gradle-wrapper.properties') }}"
+        restore-keys: "${{ runner.os }}-gradle-"
```

Comment:
The setup-gradle task should handle this already. I'm not sure why this is separately needed.

Reply:
I read somewhere that this is better, but I'll remove it